masterlist.yaml

```diff
@@ -12828,8 +12826,7 @@ plugins:
     inc: [ 'DragonBridgeBoardwalk.esp' ]
     tag:
       - Actors.AIPackages
-      - -Invent
-      - Invent.Add
```

Comment:
You can keep the the `Invent.Add`. It won't make a difference in WB, but it might prompt the author to replace the `Invent` tag.